CHANGELOG.md

```diff
@@ -5,36 +5,38 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/ACWI-SSWD/nldi-services/compare/nldi-services-1.2.0...master)
-  * /basin request now returns a 404 instead of 500 error when basin is not found (WDFN-334)
-
+* /basin request now returns a 404 instead of 500 error when basin is not found (WDFN-334)
```

Comment:
Codacy wants you to add two additional spaces after the * before the content of the bullet. I think your previous attempts was adding spaces before the bullet indicator and that's not what it wants to see.

Reply:
Fixed now for all files.